phac-nml/irida-next
# Fix for memberships on group transfer job

## What does this PR do and why?
The job fixes the memberships after a group transfer, so that the access level is ascending through its descendants.
Project transfers is out of scope in this PR.
Fixes part of https://github.com/phac-nml/irida-next/issues/226.

## Screenshots or screen recordings
N/A

## How to set up and validate locally
1. Verify the UpdateMembershipsJob is called when a group transfer is initiated. 
  a. Please see instructions from #264
2. Call the UpdateMembershipsJob from console.
  a. Open a rails console. Run `rails c` in a terminal.
  b. Call `UpdateMembershipsJob.perform_now` to start a job. Pass it a list of membership ids.
  c. Verify the memberships were updated.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
